java-checks/src/test/files/checks/security/SecureXmlTransformerCheck.java

```diff
@@ -0,0 +1,88 @@
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.XMLConstants;
+
+class A {
+
+  TransformerFactory classField = TransformerFactory.newInstance();
+
+  TransformerFactory no_call_to_securing_method() {
+    TransformerFactory factory = TransformerFactory.newInstance(); // Noncompliant
+    return factory;
+  }
+
+  // setFeature
+
+  TransformerFactory secure_processing_true() {
+    TransformerFactory factory = TransformerFactory.newInstance();
+    factory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
```

Comment:
would be useful to have a test where `true` is in a constant

Reply:
Added in eb6a439ed